string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,19 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  switch (char) {
+    case 'A':
+    case 'a':
+    case 'E':
+    case 'e':
+    case 'I':
+    case 'i':
+    case 'O':
+    case 'o':
+    case 'U':
+    case 'u':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
We can actually simplify this even more by converting `char` to lowercase before we check. That way you don't need to check for both uppercase and lowercase.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```